packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,118 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('and a response example is defined, but no response example is requested', () => {
+          it('returns the first example', () => {
+            const response = httpMocker.mock({
+              input: mockInput,
+              resource: mockResource,
+              config: { mock: { dynamic: false } },
+            });
+
+            expect(response.body).toBeDefined();
+
+            const selectedExample = flatMap(mockResource.responses, res =>
+              flatMap(res.contents, content => content.examples || []),
+            )[0];
+
+            expect(selectedExample).toBeDefined();
+            expect(response.body).toEqual((selectedExample as INodeExample).value);
+          });
+        });
+
+        describe('and the resource has no response examples', () => {
+          const resourceOperation: IHttpOperation = {
+            id: 'id',
+            method: 'get',
+            path: '/test',
+            request: {},
+            responses: [
+              {
+                code: '200',
+                headers: [],
+                contents: [
+                  {
+                    mediaType: 'application/json',
+                    schema: {
+                      type: 'object',
+                      properties: {
+                        name: { type: 'string', examples: ['Clark'] },
```

Comment:
I think you're missing examples: [] and examples: ['a', 'b']

Reply:
Can you elaborate? I fail to understand

Reply:
c7896e8